src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -0,0 +1,18 @@
+<%@ page contentType="text/html;charset=UTF-8" %>
+<html>
+<head>
+    <title>Index page</title>
+</head>
+<body>
+<h1>Hallo, mates!</h1>
+<h2>Information is written to our resource</h2>
+<p>
+    <%@ page import="java.time.LocalDateTime" %>
+    <%
+        LocalDateTime now = LocalDateTime.now();
+        String info = "Information wrote: " + now;
+    %>
+    <%= info %>
+</p>
```

Comment:
don't use java code inside html. if you want to pass smth to jsp, use request parameters. but we will do this in next homeworks